src/main/java/com/cloudbees/jenkins/support/util/IgnoreCloseOutputStream.java

```diff
@@ -28,21 +28,36 @@
 import org.kohsuke.accmod.restrictions.NoExternalUse;
 
 import javax.annotation.Nonnull;
-import java.io.FilterOutputStream;
+import java.io.BufferedOutputStream;
 import java.io.IOException;
 import java.io.OutputStream;
 
 /**
- * Provides a {@link FilterOutputStream} that ignores calls to close its underlying stream and instead simply flushes it.
+ * Provides a {@link BufferedOutputStream} that ignores calls to close its underlying stream and instead simply flushes it.
  *
  * @since TODO
  */
 @Restricted(NoExternalUse.class)
-public final class IgnoreCloseOutputStream extends FilterOutputStream implements WrapperOutputStream {
-    public IgnoreCloseOutputStream(@Nonnull OutputStream out) {
+public final class IgnoreCloseOutputStream extends BufferedOutputStream implements WrapperOutputStream {
+    /**
+     * The underlying output stream to be filtered.
+     *
+     * @param out the underlying output stream.
+     */
+    public IgnoreCloseOutputStream(OutputStream out) {
         super(out);
     }
 
+    /**
+     * The underlying output stream to be filtered.
+     *
+     * @param   out    the underlying output stream.
+     * @param   size   the buffer size.
+     */
+    public IgnoreCloseOutputStream(OutputStream out, int size) {
```

Comment:
Is it created to be used some way? If not, better to keep it simple. We can create this one when the need comes